layer/support/layer_utils.cc

```diff
@@ -18,15 +18,15 @@
 #include <cstdint>
 
 #ifdef __linux__
-#include <sys/types.h>
+#include <sys/syscall.h>
```

Comment:
You're replacing `<sys/types.h>`. Is this intended?

Reply:
Yes.

Reply:
Why? `<sys/types.h>` is not where `gettid` was coming from.

Reply:
@vettoreldaniele `types.h` is not actually used anywhere in this file

Reply:
Ok so just a cleanup, that answers my question

Reply:
So what conceptually happened is that Milad added a new header and separately dropped an unnecessary one.